Shared/Samples/List spatial reference transformations/ListSpatialReferenceTransformationsView.Model.swift

```diff
@@ -40,57 +40,23 @@ extension ListSpatialReferenceTransformationsView {
-            get { graphicsOverlay.graphics.last?.geometry }
-            set { graphicsOverlay.graphics.last?.geometry = newValue }
+            get { graphicsOverlay.graphics.last!.geometry }
+            set { graphicsOverlay.graphics.last!.geometry = newValue }
         }
         
         /// The list of transformations suitable for projecting between the original geometry's and the map's spatial references.
-        @Published private(set) var transformations: [DatumTransformation] = []
+        @Published private(set) var transformations: [GeographicTransformation] = []
         
         /// The transformation selected by the user.
-        @Published private(set) var selectedTransformation: DatumTransformation?
-        
-        /// The missing Projection Engine filenames for the selected transformation.
-        @Published var missingFilenames: [String]?
+        @Published private(set) var selectedTransformation: GeographicTransformation?
         
         // MARK: Methods
         
-        /// Updates the transformations list using the transformation catalog.
-        /// - Parameter extent: The bounding box of coordinates to be transformed.
-        func updateTransformationsList(withExtent extent: Envelope? = nil) {
-            // Get the input and output spatial references.
-            guard let inputSpatialReference = Geometry.originalGeometry.spatialReference,
-                  let outputSpatialReference = map.spatialReference
-            else { return }
-            
-            // Get the transformations from the transformation catalog.
-            transformations = TransformationCatalog.transformations(
-                from: inputSpatialReference,
-                to: outputSpatialReference,
-                areaOfInterest: extent
-            )
-            
-            // Remove the current transformation selection if it is not in the new list.
-            guard let selectedTransformation, !transformations.contains(selectedTransformation)
-            else { return }
-            
-            self.selectedTransformation = nil
-            projectedGeometry = nil
-        }
-        
         /// Selects a given transformation and projects the geometry accordingly.
         /// - Parameter transformation: The transformation.
-        func selectTransformation(_ transformation: DatumTransformation) {
-            // Remove the selection and show an alert if the transformation is missing files.
-            if transformation.isMissingProjectionEngineFiles {
-                missingFilenames = transformation.missingProjectionEngineFilenames
-                projectedGeometry = nil
-                selectedTransformation = nil
-                return
-            }
-            
+        func selectTransformation(_ transformation: GeographicTransformation) {
             // Project the original geometry using the transformation.
             guard let outputSpatialReference = map.spatialReference else { return }
```

Comment:
```suggestion
            let outputSpatialReference = map.spatialReference!
```
To keep consistent as line 98.